Firmware/language_pt.h

```diff
@@ -0,0 +1,343 @@
+/**
+ * Portuguese
+ *
+ * LCD Menu Messages
+ * Please note these are limited to 17 characters!
+ *
+ */
+
+#define(length=20) WELCOME_MSG              CUSTOM_MENDEL_NAME " pronta."
+#define MSG_SD_INSERTED                     "Cartao inserido"
+#define MSG_SD_REMOVED                      "Cartao removido"
+#define MSG_MAIN                            "Principal"
+#define MSG_DISABLE_STEPPERS                "Desativar motores"
+#define MSG_AUTO_HOME                       "Pos. de origem"
+#define MSG_SET_HOME_OFFSETS                "Def desvio origem"
+#define MSG_SET_ORIGIN                      "Definir origem"
+#define MSG_COOLDOWN                        "Arrefecer"
+#define MSG_SWITCH_PS_ON                    "Ligar alimentaçao"
+#define MSG_SWITCH_PS_OFF                   "Desligar aliment."
+#define MSG_MOVE_AXIS                       "Mover eixos"
+#define MSG_MOVE_X                          "Mover X"
+#define MSG_MOVE_Y                          "Mover Y"
+#define MSG_MOVE_Z                          "Mover Z"
+#define MSG_MOVE_E                          "Extrusora"
+#define MSG_SPEED                           "Velocidade"
+#define MSG_NOZZLE                          "Bocal"
```

Comment:
Marlin "Bico"?

Reply:
"Bico" has a special meaning in Portugal.
Roughly it is a X-Rated word... but yes, we can use it.
I'll edit that.